maxim/logger/livekit/job_context.py

```
@@ -0,0 +1,50 @@
+    try:
+        scribe().debug(
+            f"[Internal][{self.__class__.__name__}] {hook_name} completed; result={result}"
+        )
+
+    except Exception as e:
+        scribe().error(
+            f"[{self.__class__.__name__}] {hook_name} failed; error={str(e)}\n{traceback.format_exc()}"
+        )
+
+
+def instrument_job_context(orig, name):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)